src/pim-types.ts

```diff
@@ -230,6 +230,16 @@ export class TaskType extends EventType {
     return this.dueDate && moment(this.dueDate.toJSDate()).isSameOrBefore(moment(), 'day');
   }
 
+  get overdue() {
+    if (!this.dueDate) {
+      return false;
+    }
```

Comment:
Maybe I'm just tired, but this piece of code looked so very confusing to me.
First of all, having the separate ifs here instead of `if ... else if ... else`.
Or even better, move the shared parts a bit above, so:
```
const dueDate = moment(this.dueDate.toJSDate());
const now = moment();
return (this.dueDate.isDate) ? dueDate.isBefore(now, 'day') : dueDate.isBefore(now);
```

Reply:
Though, coming to think about it, I think that this condition is not really needed. If it has no time, the time is automatically 00:00 and then before, is the same as before(..., 'day).
So just change the whole thing to:

```
return (this.dueDate) ? moment(this.dueDate.toJSDate()).isBefore(moment()) : false;
```

Reply:
But then this marks tasks overdue right when the day they are due starts. Like if a task is due on April 1, it will be be overdue on April 1. Is this how we want it to work? I would think that a due date without a time shouldn't be overdue until the day after.

Reply:
Ah, and the "day" variant doesn't behave this way? Sure then, keep the differentiation then, though my first comment still applies.